app/controllers/project/messages/new.js

```diff
@@ -0,0 +1,7 @@
+import Controller from '@ember/controller';
+import { alias } from '@ember/object/computed';
+
+export default Controller.extend({
+  message: alias('model'),
+  user: alias('model.conversations.firstObject.user')
```

Comment:
This, like many other places in the code, assumes a single user per message, meaning a single conversation as well.

As we add UI support for multiple users, we will need to update this.